File: code-coaches/easy/gotham-city/GothamCity.java
import java.util.Scanner;

public class GothamCity {

    public static String isBackupNeeded(int criminals) {
        if (criminals < 5)
            return "I got this!";
        else if (criminals >= 5 && criminals <= 10)
            return "Help me Batman";
        else
            return "Good Luck out there!";
    }

	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		int criminals = input.nextInt();

		System.out.println(isBackupNeeded(criminals));

		input.close();
	}

}

File: code-coaches/easy/gotham-city/gotham_city.py
def is_backup_needed(criminals: int) -> str:
    if criminals < 5:
        return "I got this!"
    elif 5 <= criminals <= 10:
        return "Help me Batman"
    elif criminals > 10:
        return "Good Luck out there!"


def main():
    criminals = int(input())
    print(is_backup_needed(criminals))


if __name__ == "__main__":
    main()
